plugins/versionpress/src/Initialization/Initializer.php

```diff
@@ -635,4 +637,27 @@ private function getEntitiesFromDatabase($entityName)
 
         return $this->database->get_results("SELECT * FROM {$this->getTableName($entityName)}", ARRAY_A);
     }
+
+    private function installComposerScripts()
+    {
+        $composerJsonPath = VP_PROJECT_ROOT . '/composer.json';
+        if (!file_exists($composerJsonPath)) {
+            return;
+        }
+
+        $composerJson = json_decode(file_get_contents($composerJsonPath));
+
+        if (!isset($composerJson->scripts)) {
+            $composerJson->scripts = new \stdClass();
+        }
+
+        if ((!isset($composerJson->scripts->{'pre-update-cmd'}) || $composerJson->scripts->{'pre-update-cmd'} === '') &&
+            (!isset($composerJson->scripts->{'post-update-cmd'}) || $composerJson->scripts->{'post-update-cmd'} === '')
+        ) {
+            $composerJson->scripts->{'pre-update-cmd'} = 'wp vp-composer prepare-for-composer-changes';
+            $composerJson->scripts->{'post-update-cmd'} = 'wp vp-composer commit-composer-changes';
```

Comment:
What if user does not have `wp-cli` installed ? how will `pre-update-cmd` and `post-update-cmd` commands behave ?